### PR TITLE
Add first interaction bot

### PR DESCRIPTION
**Pull request by berrydenhartog**
# Description

Included a first interaction bot to notify new contributors of the contribute guidelines

### CODE REVIEW

**Comment by berrydenhartog** [comment]
pre-commit.ci run

File: .github/workflows/first-interaction.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: |
+              Welcome to our community :hugs: and thank you for your first contribution.
+
+              As a first time contributor please make sure to review our [contribution guidelines](https://github.com/MinBZK/ai-validation/blob/master/CONTRIBUTE.md) :heart:
```
Comment (robbertbos) [inline .github/workflows/first-interaction.yml:15]:
```suggestion
              As a first time contributor please make sure to review our [contribution guidelines](https://github.com/MinBZK/ai-validation/blob/main/CONTRIBUTE.md) :heart:
```

File: .github/workflows/first-interaction.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        pr-message: |
+              Welcome to our community :hugs: and thank you for your first contribution.
+
+              As a first time contributor please make sure to review our [contribution guidelines](https://github.com/MinBZK/ai-validation/blob/master/CONTRIBUTE.md) :heart:
```
Comment (robbertbos) [inline .github/workflows/first-interaction.yml:19]:
```suggestion
              As a first time contributor please make sure to review our [contribution guidelines](https://github.com/MinBZK/ai-validation/blob/main/CONTRIBUTE.md) :heart:
```

**Review by robbertbos — CHANGES_REQUESTED** [review]

File: .github/workflows/first-interaction.yml (hunk 2 above)
Comment (berrydenhartog) [reply .github/workflows/first-interaction.yml:19]: Nice catch

**Review by robbertbos — APPROVED** [review]
Approved, but not merged yet because of a dependency with https://github.com/MinBZK/ai-validation/pull/75

**Pull request merged**